tasks/maas_host_monitoring.yml

```diff
@@ -13,15 +13,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-- name: Playbook for deploying horizon
-  hosts: horizon_all
-  user: root
-  gather_facts: true
-  pre_tasks:
-    - include: common/create-grant-db.yml
-      db_name: "{{ horizon_galera_database }}"
-      db_password: "{{ horizon_container_mysql_password }}"
-  roles:
-    - role: "{{ horizon_rolename | default('os_horizon') }}"
-  vars_files:
-    - common/test-vars.yml
\ No newline at end of file
+- include: cdm.yml
+
+# NOTE(Cloudnull): This will load only the one vendor specific host file
+#  and fail if not found when host_check is true
+- include: "{{ ansible_system_vendor.split()[0] | lower }}.yml"
```

Comment:
Are you sure the facts are gathered beforehand?

Else we need a setup task.

Also that would be good to explain why you split() [on all whitespaces?] instead of split(" "), for posterity.